pages/index.html

```diff
@@ -17,112 +17,87 @@
 <div class="frameworkcontent">
-  <p>You can download <a href="<#CURRENTVERSIONURL>">the current stable version,
-      Coq <#CURRENTVERSION></a>, released in <#CURRENTVERSIONDATE>.</p>
-<#ifdef UPCOMINGVERSION>
-  <p><a href="<#UPCOMINGVERSIONURL>">The upcoming version, Coq <#UPCOMINGVERSION></a>
-    is also available for testing.</p>
-</#ifdef>
-<p>
-To use Coq, you will need a user interface.  Many editor support
-extensions are available (for Emacs, Vim, VSCode, etc.), as well as
-CoqIDE, a standalone IDE for Coq.
-The <a href="user-interfaces.html">User interfaces</a> page provides a
-full list, with more details.  If you are only looking for a quick way
-to try Coq without installing anything, we recommend you
-use <a href="https://jscoq.github.io">jsCoq</a>.
-</p>
+<p>You don't need to install Coq to get started: run Coq in your browser using
+<a href="https://jscoq.github.io">jsCoq</a>!</p>
+
+<p>To go beyond, you can proceed to <a href="/download">install the
+Coq platform</a> on your machine and
+a <a href="user-interfaces.html">user interface</a> of your
+choice.</p>
 </div>
 <div class="frameworklinks">
 <ul>
-<li><a href="<#CURRENTVERSIONURL>">Get Coq <#CURRENTVERSION></a></li>
-<#ifdef UPCOMINGVERSION>
-<li><a href="<#UPCOMINGVERSIONURL>">Get Coq <#UPCOMINGVERSION> (testing)</a></li>
-</#ifdef>
-<li><a href="https://jscoq.github.io">Try jsCoq</a></li>
+<li><a href="https://jscoq.github.io">Try Coq in your browser</a></li>
+<li><a href="/download">Install the Coq platform</a></li>
 <li><a href="user-interfaces.html">User interfaces</a></li>
 </ul>
 </div>
 </div>
 
 <div class="framework">
-<div class="frameworklabel"><a  style="color:black;font-weight:bold" href="https://github.com/coq/coq/blob/master/CONTRIBUTING.md">How to contribute?</a></div>
+<div class="frameworklabel"><a style="color:black;font-weight:bold" href="/community.html">User Community</a></div>
 <div class="frameworkcontent">
-    <p>You can contribute to the development of Coq by reporting bugs,
-    submitting pull requests, improving the documentation, and in many other
-    ways. We are also looking for sponsors who want to contribute financially
-    through the <a href="/consortium">Coq Consortium</a>. If you are interested,
-    please get in touch!</p>
-</div>
 
-<div class="frameworklinks">
-<ul>
-<li><a href="/bugs/">Bug tracker</a></li>
-<li><a href="https://github.com/coq/coq/blob/master/CONTRIBUTING.md">Contributing guide</a></li>
-<li><a href="/consortium">Coq Consortium</a></li>
-</ul>
-</div>
-</div>
+<p>Coq has a strong and active user community. They are contributing
+formal developments, extensions of Coq
+(see <a href="/packages.html">Coq Package Index</a>), and tools based
+on Coq (see <a href="/related-tools.html">Related Tools</a>).  We have
+a (multi-lingual) <a href="https://coq.discourse.group">Discourse
+forum</a> and a <a href="https://coq.zulipchat.com">Zulip chat</a>.
+</p>
```

Comment:
>"formal developments" - you mean proofs, definitions, ?? The term doesn't do much for me.

It's not my words, but yes formal definitions and proofs.

>"tools based on Coq" - includes both tools built on top of Coq and tools for use with Coq IIUC, so that name isn't a precise fit.

Right. Any better suggestion?

>Is it worth suggesting how to reach individuals? Or seeing a list of the main contributors? OTOH some prefer to be less visible.

It would indeed be a good thing to get a team page. This is something that has been wanted for a long time. But much beyond the scope of this PR.

>Also, BTW, the Related Tools page has a lot of bullet points that start with "XXX is a/an YYYY". Would be less repetitive to change these to "XXX - a YYY"

Also, this page would be worth keeping more up-to-date. I'm thinking maybe a manually curated page is not a good fit for this.

Reply:
How about "tools for use with Coq and based on Coq"?